libbeat/docs/upgrading.asciidoc

```diff
@@ -4,214 +4,142 @@
+To check which version of the template is loaded, open Kibana Console, call `GET
+/_template/metricbeat`, and look for the version string. Note that you need to
+do this for each Beat type that you are running (e.g. Filebeat, Metricbeat,
+Packetbeat).
 
-==== Filebeat registry file location and format
-
-Filebeat stores the read offset in a registry file that it persists to disk.
-The format of this file has changed in version 5.0, but Filebeat automatically
-upgrades the format on the first run, so there's normally no action required on
-upgrading.
-
-If you have installed Filebeat from the DEB or RPM packages, the location of the
-file stays the same as well (`/var/lib/filebeat/registry`). Also if you've
-installed Filebeat as a service on Windows, the location stays the same
-(`C:\ProgramData\filebeat\registry`).
+==== Migrate configuration files
 
-If you are running Filebeat from the `tar.gz` archive, however, the location of
-the registry file has changed. In version 1.x it was by default named
-`.filebeat` in the current working directory. In version 5.0, it is
-`data/registry` relative to the binary location. For upgrading in this case, we
-recommend copying the `.filebeat` file to `data/registry`.
+Beats 6.0 comes with several backwards incompatible configuration changes.
+Please review the <<breaking-changes-6.0>> document. Where possible, we kept the
+old configuration options working but deprecated. However, that is not always
+possible, so if you use any of the settings mentions in the Breaking Changes
+section of the release notes, make sure to understand the alternatives that we
+provide.
 
-==== Upgrade Topbeat to Metricbeat
+===== modules.d configuration layout
 
-NOTE: When upgrading to Metricbeat, keep in mind that any data you've collected
-with Topbeat is not compatible with the 5.0 version of the Beats dashboards
-because the underlying event data structures have changed.
+Starting with the 6.0 version, Filebeat and Metricbeat are moving to a directory
+layout for configuration, where each module is configured in its own
+configuration file.
 
-With the Beats 5.0 release, Topbeat is replaced by Metricbeat, which offers more
-functionality. More precisely, the Topbeat functionality is roughly equivalent
-to the "system" module in Metricbeat.
+While the all-in-one configuration is still fully supported, we recommend moving
+to the new layout at upgrade time. This typically means starting of with the new
+default configuration and modify it with the custom settings that you had in
+your old files.
 
-While there is no automatic way to convert a Topbeat configuration file to the
-equivalent of a Metricbeat configuration file, both configurations are simple
-enough that migrating shouldn't take long.
+===== New command `test config` command
 
-For example, this is the input section of the Topbeat configuration file:
+Beats 6.0 introduce a new command to test the configuration file, via the `test`
```

Comment:
Since you are using Beats 6.0 as a product name, use the singular form of the verb. I'd also condense this a bit and say: 

Beats 6.0 introduces a new `test` command for testing the configuration file. For example: